dev/io.openliberty.org.jboss.resteasy.common.ee11/src/org/jboss/resteasy/core/ContextParameterInjector.java

```diff
@@ -27,18 +27,19 @@
 import java.lang.reflect.Method;
 import java.lang.reflect.Proxy;
 import java.lang.reflect.Type;
-import java.security.AccessController;
-import java.security.PrivilegedAction;
 import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.CompletionStage;
+import java.security.AccessController; //Liberty Change
+import java.security.PrivilegedAction; //Liberty Change
```

Comment:
```suggestion
```
This should not be needed.  EE 11 doesn't support SecurityManager.  Any reference should be removed.